cypress/integration/form.js

```diff
@@ -75,4 +78,46 @@ context('Form Blocks Tests', () => {
     cy.get('#page-document button.submit-block-button').click();
   });
 
+  it('As editor I can add a multi-line text field to a form', () => {
+    cy.get('.block.inner.text .public-DraftEditor-content').click();
+    cy.get('button.block-add-button').click();
+    cy.findByText('Form').click();
+    cy.findByText('Textarea').click();
+    cy.get('input[id=field-textarea-label]').type('My name');
+    cy.get('#toolbar-save').click();
+    cy.url().should('eq', Cypress.config().baseUrl + '/my-page');
+    cy.findByText('My name').should('exist');
```

Comment:
@iFlameing please be more specific about what you test here. Try to look up the field and check if a label with "My name" exists and a textarea.

Reply:
@tisto I have added the specific test for the label field but just one thing `.findByText` searches the text present in the document and if we are able to find that we can confirm that our textarea rendered because we are passing label value to TextareaWidget as title props.